src/Microsoft.Framework.Logging/Utils/TypeHelper.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
last minute change so missed the classname. The namespace is consistent with all the files in that folder